src/components/User/Badges/BadgesSidebar.tsx

```diff
@@ -0,0 +1,83 @@
+import React, { useState } from 'react'
+
+import useFormatMessage from 'decentraland-gatsby/dist/hooks/useFormatMessage'
+import { Close } from 'decentraland-ui/dist/components/Close/Close'
+import Sidebar from 'semantic-ui-react/dist/commonjs/modules/Sidebar/Sidebar'
+
+import { Badge, UserBadges } from '../../../entities/Badges/types'
+import ChevronLeft from '../../Icon/ChevronLeft'
+
+import BadgeCard from './BadgeCard'
+import BadgeDetail from './BadgeDetail'
+import './BadgesSidebar.css'
+
+interface Props {
+  isSidebarVisible: boolean
+  onClose: () => void
+  badges: UserBadges
+}
+
+export default function BadgesSidebar({ isSidebarVisible, onClose, badges }: Props) {
+  const t = useFormatMessage()
+  const [badgeInDetail, setBadgeInDetail] = useState<Badge | null>(null)
+
+  const handleClose = (e: React.MouseEvent<unknown>) => {
+    e.preventDefault()
+    e.stopPropagation()
+    onClose()
+  }
+
+  return (
+    <Sidebar
+      className="BadgesSidebar"
```

Comment:
Can't you reuse the same Sidebar used in CompetingProposals? It can be a Common component.